src/EnergyPlus/MixedAir.cc

```diff
@@ -3714,7 +3714,11 @@ namespace MixedAir {
             auto &curAirLoopFlow(AirLoopFlow(AirLoopNum));
 
             curAirLoopFlow.OAMinFrac = OutAirMinFrac;
-            curAirLoopFlow.MinOutAir = OutAirMinFrac * this->MixMassFlow;
+            if (this->FixedMin) {
+                curAirLoopFlow.MinOutAir = min(OutAirMinFrac * curAirLoopFlow.DesSupply, this->MixMassFlow);
+            } else {
+                curAirLoopFlow.MinOutAir = OutAirMinFrac * this->MixMassFlow;
+            }
```

Comment:
OK, so this section just constrains the minimum outside air to the current mixed air flow rate.